packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -2541,12 +2570,14 @@ describe("Runtime", () => {
 			assert.equal(mockLogger.events.length, 0, "Expected no more events logged");
 		});
 
-		describe("Signals", () => {
+		describe("Signal Telemetry", () => {
 			let containerRuntime: ContainerRuntime;
 			let logger: MockLogger;
-			let droppedSignals: ISignalEnvelope[];
+			let droppedSignals: ISignalEnvelopeWithClientIds[];
+			let runtimes: Map<string | undefined, ContainerRuntime>;
```

Comment:
Map with `undefined` key? That is unexpected and I am super curious if it actually works. Sure we don't want to avoid that? Use some other sentinel for place where you deal with undefined?